documentation/onboard-dpg-in-sdkautomation/README.md

```diff
@@ -0,0 +1,41 @@
+# Service Onboard DPG with Swagger CI Pipeline
+
+Service Onboarding DPG with Swagger CI pipeline can help you get generated SDK and ApiView during the procedure of creating swagger PR.
+
+This Document describes the process of how to onboard DPG with SDK automation pipeline. __This document is targeting for service team.__ 
+
+*If you want to go through details of pipeline implementation, please go to [Integrate DPG and ApiView in SDK Automation Pipeline](Integrate-dpg-and-apiview-in-sdk-automation-pipeline.md).*
+
+## WorkFlow
+
+![workflow-for-service-team](workflow-service-team.png)
```

Comment:
It looks i can not comment on the pic directly so add the comments here:

1,   The first step should be create Swagger PR instead of 'Swagger PR'
2.   For 'create branch', do you need to specify the this is about the SDK repo?  And what if service team do not have permissions?
3.   Who will be responsible for deleting the branch?

Reply:
For the permission issue, there is a FAQ in the doc, and it tells services who they can ask for help. For others, I will change/add the related information